Cargo.lock

```diff
@@ -450,7 +450,7 @@ dependencies = [
 
 [[package]]
 name = "deno_core"
-version = "0.60.0"
```

Comment:
`deno_core` + `deno_web` + `deno_fetch` + `std` don't need a minor bump right? Patch should be enough.

Reply:
CI is setup to release those crates on tag, it's just easier to do that than releasing just cli.

Reply:
I'm not saying we shouldn't bump them. Im saying instead of incrementing the minor version, we could increment the patch version. Doesn't really matter though.